calc/src/mechanics/gen789.ts

```diff
@@ -1049,8 +1049,7 @@ export function calculateSMSSSV(
   }
 
   if (field.chromaticField === 'Forgotten-Battlefield') {
-    if ((move.named('Poltergeist') && (!defender.item || isQPActive(defender, field))) || // Forgotten Battlefield - Poltergeist never fails
-        (move.named('Gigaton Hammer') && defender.hasType('Steel'))) { // Forgotten Battlefield - Gigaton Hammer deals neutral damage to steel types
+    if ((move.named('Gigaton Hammer') && defender.hasType('Steel'))) { // Forgotten Battlefield - Gigaton Hammer deals double damage to steel and fighting types
```

Comment:
polish: remove unnecessary parentheses
```suggestion
    if (move.named('Gigaton Hammer') && defender.hasType('Steel')) { // Forgotten Battlefield - Gigaton Hammer deals double damage to steel and fighting types
```